pkg/notification/db_changes.sql

```diff
@@ -19,33 +19,34 @@
 -- This file can be used to make the required changes to the MySQL DB. This is
 -- not a proper migration but it should work on most situations.
 
-CREATE TABLE `cbox_notifications` (
+CREATE TABLE `notifications` (
 	`id` INT PRIMARY KEY AUTO_INCREMENT,
 	`ref` VARCHAR(3072) UNIQUE NOT NULL,
 	`template_name` VARCHAR(320) NOT NULL
 );
 
 COMMIT;
 
-CREATE TABLE `cbox_notification_recipients` (
+CREATE TABLE `notification_recipients` (
 	`id` INT PRIMARY KEY AUTO_INCREMENT,
 	`notification_id` INT NOT NULL,
 	`recipient` VARCHAR(320) NOT NULL,
 	FOREIGN KEY (notification_id)
-		REFERENCES cbox_notifications (id)
+		REFERENCES notifications (id)
 		ON DELETE CASCADE
 );
 
 COMMIT;
 
-CREATE INDEX `cbox_notifications_ix0` ON `cbox_notifications` (`ref`);
+CREATE INDEX `notifications_ix0` ON `notifications` (`ref`);
 
-CREATE INDEX `cbox_notification_recipients_ix0` ON `cbox_notification_recipients` (`notification_id`);
-CREATE INDEX `cbox_notification_recipients_ix1` ON `cbox_notification_recipients` (`user_name`);
+CREATE INDEX `notification_recipients_ix0` ON `notification_recipients` (`notification_id`);
+CREATE INDEX `notification_recipients_ix1` ON `notification_recipients` (`recipient`);
 
 -- changes for added notifications on oc shares
 
 ALTER TABLE cernboxngcopy.oc_share ADD notify_uploads BOOL DEFAULT false;
+ALTER TABLE cernboxngcopy.oc_share ADD notify_uploads_extra_recipients VARCHAR(2048);
```

Comment:
Great! Still this one refers to the DB name...

Reply:
Fixed. As long as the script is running with the proper database selected beforehand, that shouldn't be needed.

Reply:
That's correct, and the script must be generic - the choice of the db is for the person deploying the script!
But.. did you push? :)

Reply:
I did, but to the test branch :laughing:  Pushed here now.